packages/devcmd/src/process/ProcessExecutor.ts

```diff
@@ -144,14 +158,25 @@ export class ProcessExecutor {
    * @returns A promise resolving to a list of exit status infos, one for each process in
    * the same order as `processList`, once all processes have exited
    */
-  async execPipedParallel(processList: ProcessInfo[]): Promise<NodeExitInfo[]>;
+  async execPipedParallel(processList: ProcessInfo[], options?: ExecPipedParallelOptions): Promise<NodeExitInfo[]>;
 
   /** Implementation for the two overload signatures above. */
   async execPipedParallel(
-    processMapOrList: ProcessInfo[] | { [id: string]: ProcessInfo }
+    processMapOrList: ProcessInfo[] | { [id: string]: ProcessInfo },
+    options?: ExecPipedParallelOptions
   ): Promise<NodeExitInfo[] | { [id: string]: NodeExitInfo }> {
-    const processEntries = Object.entries(processMapOrList);
-    this.logger.notice(`Beginning parallel execution of ${processEntries.length} processes...`);
+    const suppressNotices = !!options?.suppressNotices;
+    const logger = this.logger.withSuppression(suppressNotices);
+
+    let processEntries = Object.entries(processMapOrList);
+    if (suppressNotices) {
+      processEntries = processEntries.map(([k, processInfo]) => {
+        const newProcessInfo = { ...processInfo, options: { ...processInfo.options, suppressNotices } };
```

Comment:
We might consider using `options: { suppressNotices, ...processInfo.options} }`, then it would be possible for one parallel command to have notices enabled...

Reply:
Ohhh, I like this suggestion! Will apply 👍